src/lib.rs

```diff
@@ -246,24 +276,7 @@ pub(crate) fn nix_err(err: nix::Error) -> io::Error {
     }
 }
 
-#[cfg(test)]
-mod test {
-    use std::mem::MaybeUninit;
-
-    #[test]
-    fn align_to_mut_is_sane() {
```

Comment:
This removed test is to make sure the assumptions around the `align_to_mut` call in `fill_events` are correct. Did it break?

Reply:
Oh, wait, I meant to move away from align_to_mut since `std::mem::size_of_val()` can handle all the size conversions for us when we're just casting to `char*`.